tools/windows_buildenv.bat

```diff
@@ -25,6 +21,10 @@ IF NOT DEFINED INSTALL_ROOT (
     SET INSTALL_ROOT=%MIXXX_ROOT%\install
 )
 
+SET BUILDENV_NAME=mixxx-deps-2.4-x64-windows-8f8342a
+SET BUILDENV_BRANCH=2.4-rel
```

Comment:
You can still use the 2.4 branch for debugging on windows. Just download and replace the folder. 
The 2.4-rel branch is the default for GitHub workflows because the debug libraries are not used there.  
This means we get exact the same binary result if we use the 2.4 branch or the 2.4-rel branch.

Reply:
Why do we need two source branches in Git to build two sets of binaries?

Reply:
Because the triplets files have the same name, but different content. https://github.com/mixxxdj/vcpkg/commit/c2c93b5d8eb42be0bd3a947e1e12b9d9a806c248
We need also a different branch so that the packages are going into two locations named by the branch on https://downloads.mixxx.org/dependencies/2.4-rel/

Reply:
If it's just the value of this one CMake variable `VCPKG_BUILD_TYPE`, please set it from outside.
Or, if this is not possible, generate the 5 files in overlay/triplets on the fly during the build of the VCPKG buildenvs.

Reply:
It is not possible to set it from the outside. This is a pending feature request for VCPKG. 

> Or, if this is not possible, generate the 5 files in overlay/triplets on the fly during the build of the VCPKG buildenvs.

But than we have still two branches. One with the edit code enabled and one without it.

Reply:
I have probably not understand what you want. 

Do you want to create the release builds and the combined debug/release build from a single workflow run?

Reply:
I request two things:
1. Compile all build environment ZIP-Fils from the same branch
2. Use the full buildenvironment ZIP-File (with Debug and Release DLLs) for local builds by default.

Reply:
1. Does the posit of the original intend. It will populate the cache during PR builds with the more code than with less. It is very convenient to develop with the release only version and reducing the build time to 3 hours instead of 5. That makes a difference.  
3. That means that every casual contributor on windows will need to download 1.3 GB instead of just 530 MB. Unpacked the environment consumes 5 GB instead of 3 GB. Why should that be the default? How about offer both options equal easy, that the user can make the choice?

Reply:
The new approach addresses 2. well!

To make it safe, please make line
https://github.com/mixxxdj/mixxx/blob/22e20ae49b728827ddec95fa0b333ce1aa57ce2f/tools/windows_buildenv.bat#L181
conditional too.

Reply:
Done.